### PR TITLE
Make copy more bland

### PR DESCRIPTION
**Pull request by artpi**
Publicize feedback messages may be a bit too flamboyant for some :)
This makes them a bit more bland

CC @gziolo @enejb

### CODE REVIEW

**Comment by enejb** [comment]
Maybe updating the copy to 
Sharing... 
Your post has been shared. 

I would avoid saying accounts because that implies a plural. Which is not correct if you only have one account that you shared it to.

File: client/my-sites/posts/post-share.jsx [hunk 1]
```diff
@@ -140,8 +140,8 @@ const PostSharing = React.createClass( {
 
 		return (
 			<div className="posts__post-share-wrapper">
-				{ this.props.requesting && <Notice status="is-warning" showDismiss={ false }>{ this.translate( 'Hang tight, socializing your media...' ) }</Notice> }
-				{ this.props.success && <Notice status="is-success" onDismissClick={ this.dismiss }>{ this.translate( 'It went out! Your social media is on fire!' ) }</Notice> }
+				{ this.props.requesting && <Notice status="is-warning" showDismiss={ false }>{ this.translate( 'Scheduling your posts...' ) }</Notice> }
+				{ this.props.success && <Notice status="is-success" onDismissClick={ this.dismiss }>{ this.translate( 'Your posts are being published. Please check your social accounts. ' ) }</Notice> }
```
Comment (omarjackman) [inline client/my-sites/posts/post-share.jsx]: Was this trailing space intentional? :smile:

**Comment by gziolo** [comment]
I agree with @enejb, you are technically sharing one post :)

**Comment by artpi** [comment]
By posts I meant Social media posts, so this is very accurate. But yes, that is confusing so its best to avoid word post.
I changed the copo once again and merging this, since its best to discuss it under call for testing.

**Pull request merged**